c10/core/DispatchKey.cpp

```diff
@@ -48,6 +48,8 @@ const char* toString(DispatchKey t) {
       return "TESTING_ONLY_GenericWrapper";
     case DispatchKey::Profiler:
       return "Profile";
+    case DispatchKey::Tracer:
+      return "Trace";
```

Comment:
This should be Tracer (exposing the bug above, where it should be Profiler, not Profile)